pkgs/swiftgen/swift2objc/lib/src/parser/_core/utils/parse_symbol_id.dart

```diff
@@ -0,0 +1,5 @@
+import '../type_defs.dart';
+
+String parseSymbolId(JsonMap symbolJson) {
```

Comment:
Don't really need a separate file for each of these functions. Can you combine `type_defs.dart`, `parse_symbol_id.dart`, and `parse_symbol_name.dart` into a single `util.dart` file?

Some of the parser files below could also be merged. I don't think you need to split these files until they get up to a few hundred or a thousand lines.

Reply:
I have a habit of splitting entities into files, though I think `utils.dart` would be sufficient in this case I guess.

Reply:
done